sdk/iot/hub/samples/src/generate_certificate.sh

```diff
@@ -22,5 +22,8 @@ echo -e "\nSample certificate generated"
 echo "Use the device_cert_store.pem file within the sample:"
 echo    export AZ_IOT_DEVICE_X509_CERT_PEM_FILE=$(pwd)/device_cert_store.pem
```

Comment:
>echo [](start = 0, length = 4)

ask them to run this command at their terminal, if they want to set the environment variable

Reply:
So we should not be setting these?

Reply:
well you can not set these variable for them, but you can print command that they should execute in the shell

Reply:
gotcha. updated